src/main/java/com/launchacademy/reviews/seeders/MainSeeder.java

```diff
@@ -0,0 +1,53 @@
+package com.launchacademy.reviews.seeders;
+
+
+import com.launchacademy.reviews.models.Venue;
+import com.launchacademy.reviews.services.VenueService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.stereotype.Component;
+
+@Component
+public class MainSeeder  implements CommandLineRunner {
+    private VenueService venueService;
+
+
+    @Autowired
+    public MainSeeder(VenueService venueService){
+        this.venueService = venueService;
+    }
+
+    @Override
+    public void run(String... args) throws Exception{
+        Venue venue1 = new Venue();
+        Venue venue2 = new Venue();
+
+        if (venueService.findAllVenues().size() <= 0){
```

Comment:
You could also set up `venueService.count()` with the repository so that you don't need to load all venues into memory to find out how many there are!